example/android/app/src/main/res/values-night/themes.xml

```diff
@@ -38,16 +38,11 @@
         <item name="kevinWindowTitleTextStyle">@style/TextAppearance.Headline3</item>
         <item name="kevinWindowSubtitleTextStyle">@style/TextAppearance.Subtitle1</item>
         <item name="kevinModalTitleTextStyle">@style/TextAppearance.ModalTitle</item>
-        <item name="kevinInputLabelTextStyle">@style/TextAppearance.Subtitle1</item>
```

Comment:
Just curious, did something changed in Android theming or is it just small example "redesign"?

Reply:
This param is removed from android SDK since it was used only for card payments/linking, and it's not needed anymore.